Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update && apt-get -y install ca-certificates
 
 ARG TARGETPLATFORM
 
-FROM --platform=$TARGETPLATFORM debian:stable-slim
+FROM --platform=$TARGETPLATFORM debian:stable-slim as release
```

Comment:
This changes what is the "default" result of running `docker build` on this repo. I think it might be good to document these two targets and its differences in the `README`.

Reply:
Right. We have some `helper` make scripts on `docker build ...` which aren't using in any automations AFAICT. The result of those would build the new final stage instead of the old one so that is worth documenting as well but shouldn't impact any of our current pipelines I think.

Will update `README`!

Reply:
README changes look good, thanks!